main.py

```diff
@@ -234,6 +234,7 @@ class App(tk.Tk):
     def __init__(self):
         tk.Tk.__init__(self)
         self.filename_label = None
+        self.mode_label = None
```

Comment:
model_label refers to which mode? There's rolling over labels or not and adding/deleting boxes. May want to break this up into 2 label widgets for each boolean.